docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2518,15 +2515,23 @@ W2 and 1099 Forms
   * - Last name
   * - Age 
   * - DOB 
-  * - Mailing Address
-  * - Mailing ZIP Code
+  * - Mailing Address Street Number 
```

Comment:
Can you add that these first three are blank if it's a PO Box, and the PO Box field is blank if it's not?